src/lib/kube_cluster.mli

```diff
@@ -25,6 +25,10 @@ val get :
         [> Storage.Error.common | `Missing_data of string | `Of_json of string ] ])
     Deferred_result.t
 
+val max_started_jobs: t -> int
+(** How many jobs should Coclobas try to start on this cluster. *)
```

Comment:
Maybe "The maximum number of jobs that Coclobas will attempt to run simultaneously on the cluster"? Since it's not the max number it'll start ever.